WordPress/Classes/ViewRelated/Reader/Select Interests/ReaderInterestsStyleGuide.swift

```diff
@@ -66,3 +66,90 @@ class ReaderInterestsStyleGuide {
         indicator.color = UIColor(light: .black, dark: .white)
     }
 }
+
+class ReaderSuggestedTopicsStyleGuide {
+    /// The array of colors from the designs
+    /// Note: I am explictly using the MurielColor names instead of using the semantic ones
+    ///       since these are explicit and not semantic colors.
+    static let colors: [TopicStyle] = [
+        // Green
+        .init(textColor: .init(colorName: .green, section: .text),
+              backgroundColor: .init(colorName: .green, section: .background),
+              borderColor: .init(colorName: .green, section: .border)),
+
+        // Blue
```

Comment:
Super small nitpick...
`// Purple` instead of `// Blue`?

Reply:
Oops, done: add5712e9dd64fd1390663a85c00448d6e339008